plugins/Makefile

```diff
@@ -689,6 +689,12 @@ PLUGIN_FILES += $(filter-out MSM/src/MSM.cpp,$(wildcard MSM/src/*.cpp))
 # modules/types which are present in other plugins
 MSM_CUSTOM = ADSR BlankPanel Delay LFO LowFrequencyOscillator Mult Noise OP VCA VCO sawTable triTable
 
+# --------------------------------------------------------------
+# Nonlinear Circuits
+
+# PLUGIN_FILES += $(filter-out nonlinearcircuits/src/NLC.cpp,$(wildcard nonlinearcircuits/src/*.cpp))
```

Comment:
this line is good, why commented out?

Reply:
oh oops. was poking around too much. will correct